modules/appeals_api/spec/support/fixture_helpers.rb

```diff
@@ -1,15 +1,34 @@
 # frozen_string_literal: true
 
+require 'forwardable'
+
 module FixtureHelpers
-  def fixture_to_s(filename, version: '')
-    File.read fixture_filepath(filename, version:)
+  extend Forwardable
+
+  def self.fixture_filepath(path_within_fixtures, version: '')
+    if version.present?
```

Comment:
There was no way to fully update the use of this helper in all files in a single PR, so I've left the signature of this method basically the same for now, but my intent is to update it so that instead of passing in a version keyword, like `fixture_filepath('headers.json', version: 'v1')` or `fixture_filepath('form.pdf', version: 'v1')`, you'd specify a subpath within the fixtures folder, like `fixture_filepath('api_name/v1/headers.json')` or `fixture_filepath('pdfs/v1/form.pdf')`.

I chose to use require a subpath within the fixtures folder as an argument instead of adding an `api_name:` keyword argument (or similar) because the PDFs aren't inherently associated with a specific API, but the JSON files are. I'd still be open to using keyword args instead of a subpath, but I switched from keyword args to a subpath while writing this because a subpath was usually shorter to type.